useScroll/useScroll.js

```diff
@@ -235,7 +235,7 @@ const useThemeScroll = (props, instances) => {
 	function handleResizeWindow () {
 		const focusedItem = Spotlight.getCurrent();
 
-		if (focusedItem) {
+		if (focusedItem && focusedItem.nodeName !== 'INPUT') {
```

Comment:
Thanks for the PR.
But this fix has still some issues in some cases.
For example, let's say there is a list that has inputs in each item. When we scroll to the middle of the list and click on the input, then the scroll position goes to the top. WIth your fix, the VKB will be still there. You can refer to the logic from https://github.com/enactjs/enact/pull/2362 which introduced this code.
Is there any event or method to notify VKB is opened so that we can prevent the issue more specifically? 

Also, have you tested on iOS devices? The issue is not reproduced on them it seems `resize` event is not firing.
Would you please check why the issue is not reproduced on iOS devices?